firestore_index_config.tf

```diff
@@ -0,0 +1,104 @@
+locals {
```

Comment:
Do the tf files need to be at the root of the repo?

Reply:
Moved the terraform to Firestore folder